lambdas/webhook_processor/README.md

```diff
@@ -0,0 +1,23 @@
+# Webhook Processor
+
+Processes webhooks by reading them from a kinesis stream and forwarding to Jenkins
```

Comment:
Perhaps we should explain a little about the purpose of this within the whole 'interceptor' design.